libraries/AP_HAL_ChibiOS/hwdef/AEROFOX-H7/hwdef.dat

```diff
@@ -0,0 +1,240 @@
+
+PD10 FRAM_CS   CS SPEED_VERYLOW
+PC14 FLASH_CS CS
+
+PE7 DRDY1_ADIS16470 INPUT GPIO(90)
+define ADIS_DRDY_PIN 90
+
+# ADC
+PA5 BATT_VOLTAGE_SENS ADC1 SCALE(1)
+PB0 BATT_CURRENT_SENS ADC1 SCALE(1)
+
+PC0 BATT2_VOLTAGE_SENS ADC1 SCALE(1)
+PC1 BATT2_CURRENT_SENS ADC1 SCALE(1)
+
+PB1 FMU_SERVORAIL_VCC_SENS ADC1 SCALE(12)
+PA4 VDD_5V_SENS ADC1 SCALE(2)
+
+# GPIO
+PA0    TIM2_CH1 TIM2 PWM(1)  GPIO(50) BIDIR
+PA1    TIM2_CH2 TIM2 PWM(2)  GPIO(51)
+PE9     TIM1_CH1 TIM1 PWM(3)  GPIO(52) BIDIR
+PE11   TIM1_CH2 TIM1 PWM(4)  GPIO(53)
+PD12  TIM4_CH1 TIM4 PWM(5)  GPIO(54) BIDIR
+PD13  TIM4_CH2 TIM4 PWM(6)  GPIO(55)
+PC6    TIM3_CH1 TIM3 PWM(7)  GPIO(56) BIDIR
+PC7    TIM3_CH2 TIM3 PWM(8)  GPIO(57)
+
+PA2    TIM15_CH1    TIM15 PWM(9)  GPIO(58)
+PA3    TIM15_CH2    TIM15 PWM(10) GPIO(59)
+PA7    TIM14_CH1   TIM14 PWM(11) GPIO(60)
+PA6    TIM13_CH1   TIM13 PWM(12) GPIO(61)
+PC9  TIM8_CH4     TIM8 PWM(13) GPIO(62)
+PC8  TIM8_CH3     TIM8 PWM(14) GPIO(63)
+PB15    TIM12_CH2     TIM12 PWM(15) GPIO(64)
+PB14    TIM12_CH1     TIM12 PWM(16) GPIO(65)
+
+PB9 TIM17_CH1 TIM17  ALARM
+
+PE3 EXTERN_GPIO1 OUTPUT GPIO(1)
+PE4 EXTERN_GPIO2 OUTPUT GPIO(2)
+PD11 EXTERN_GPIO3 OUTPUT GPIO(3)
+#PD14 EXTERN_GPIO4 OUTPUT GPIO(4)
+
+PD14 LED_SAFETY OUTPUT
+PD15 SAFETY_IN INPUT PULLDOWN
+
+PC13 VDD_3V3_SENSORS_EN OUTPUT LOW OPENDRAIN
+PC4 nVDD_5V_PERIPH_EN OUTPUT LOW OPENDRAIN
+PE8 nVDD_12V_PERIPH_EN OUTPUT HIGH OPENDRAIN
+
+PE10 VDD_5V_HIPOWER_oOC INPUT PULLUP
+PC5 VDD_BRICK_nVALID INPUT PULLUP
+
+# BARO
+BARO SPL06 I2C:0:0x76
+define AP_BACKEDN_DEFAULT_ENABLE 0
+define AP_BARO_SPL06_ENABLE 1
+
+# COMPASS
+COMPASS QMC5883L I2C:0:0x0D false ROTATION_NONE
+
+# LED setup is similar to PixRacer
+
+define AP_NOTIFY_GPIO_LED_RGB_ENABLED 1
+PE14 LED_RED OUTPUT GPIO(10)
+PE12 LED_GREEN OUTPUT GPIO(11)
+PE13 LED_BLUE OUTPUT GPIO(12)
+
+define AP_NOTIFY_GPIO_LED_RGB_RED_PIN 10
+define AP_NOTIFY_GPIO_LED_RGB_GREEN_PIN 11
+define AP_NOTIFY_GPIO_LED_RGB_BLUE_PIN 12
+
+
+# IMU heater
+PB8 HEATER_EN OUTPUT LOW GPIO(80)
+define HAL_HEATER_GPIO_PIN 80
+define HAL_HAVE_IMU_HEATER 1
+define HAL_IMU_TEMP_DEFAULT 35
+define HAL_IMUHEAT_P_DEFAULT 50
+define HAL_IMUHEAT_I_DEFAULT 0.07
+define HAL_IMU_TEMP_MARGIN_LOW_DEFAULT 5
+
+# compass
+define HAL_PROBE_EXTERNAL_I2C_COMPASSES
+
+define HAL_SKIP_AUTO_INTERNAL_I2C_PROBE
+
+define HAL_I2C_INTERNAL_MASK 1
+
+# SWD debugging
+PA13 JTMS-SWDIO SWD
+PA14 JTCK-SWCLK SWD
+
+define HAL_CHIBIOS_ARCH_FMUV3 1
+
+define BOARD_TYPE_DEFAULT 3
+
+# safety switch
+define HAL_HAVE_SAFETY_SWITCH 1
+
+# FRAM
+define HAL_WITH_RAMTRON 1
+define HAL_STORAGE_SIZE 32768
+
+# open all IMUs
+define HAL_EKF_IMU_MASK_DEFAULT 7
+
+# IMU fast sample
+define HAL_DEFAULT_INS_FAST_SAMPLE 7
+
+# Enable FAT
+define HAL_OS_FATFS_IO 1
+
+# DMA
+DMA_NOSHARE SPI2* UART2* UART4* UART5*
+
+ENABLE_DFU_BOOT 1
+
+define HAL_BATT_MONITOR 4 
+define HAL_BATT_VOLT_PIN 19 
+define HAL_BATT_CURR_PIN 9 
+define HAL_BATT_VOLT_SCALE 21 
+define HAL_BATT_CURR_SCALE 40 
+
+define HAL_BATT2_MONITOR 4 
```

Comment:
probably should not enable 2nd monitor by default...just remove line 236

Reply:
OK, I'll make the changes right away.